newbius/modules/slurm/templates/helm_values/slurm_cluster.yaml.tftpl

```diff
@@ -64,6 +64,25 @@ periodicChecks:
       thresholdMoreThan: ${ncclBenchmark.min_threshold}
 
 slurmNodes:
+  accounting:
+    enabled: ${nodes.accounting.enabled}
+    k8sNodeFilterName: ${k8s_node_filters.non_gpu.name}
+    mariadbOperator:
+      enabled: ${nodes.accounting.mariadbOperator.enabled}
+      %{~ if nodes.accounting.enabled ~}
```

Comment:
Shouldn't it depend on `nodes.accounting.mariadbOperator.enabled` rather than `nodes.accounting.enabled`?